src/common/types.hockey.ts

```diff
@@ -34,19 +34,17 @@ export type AwardPlayer = {
 	name: string;
 	tid: number;
 	abbrev: string;
+	keyStats: string;
 	pts: number;
-	trb: number;
-	ast: number;
 };
 
 export type AwardPlayerDefense = {
 	pid: number;
 	name: string;
 	tid: number;
 	abbrev: string;
-	trb: number;
-	blk: number;
-	stl: number;
+	tk: number;
+	gaa: number;
```

Comment:
gaa is for goalies, but this award is defensemen. So maybe make this `hit`? Not sure what defensive stats would be best to show. `dps` too?